assignment1.txt

```diff
@@ -0,0 +1,12 @@
+1. What is a compiler?
+A compiler is a program that transforms code written in a given language into one that the computer can  read
```

Comment:
right, more specifically, a compiler takes your code written in C, removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.